scripts/install.bat

```diff
@@ -0,0 +1,197 @@
+@ECHO OFF
```

Comment:
all the same concerns raised in the `.sh`

Reply:
Edited both the `.bat` and `.sh` files. I addressed the concerns you mentioned as best I could. All others have been commented on in their respective sections.